R/package_skeleton.R

```diff
@@ -40,18 +43,20 @@ package_skeleton <- function(path = ".", git_init = TRUE, git_clone = NULL,
   }
   
   if (description_template ) {
-    add_description()
+    add_description(npath)
   }
   
   ## Add in package setup files
-  devtools::setup(rstudio = FALSE) 
+  devtools::setup(npath, rstudio = FALSE) 
 
   ## Add the necessary R files and directories
   #message("Creating new package in ", npath)
   add_contributing(npath)
   if (CoC) add_code_of_conduct(npath, package = FALSE, coc_email = coc_email)
   add_readme(npath, package = FALSE)
   
+  add_rbuildignore(npath)
```

Comment:
Should we append bcgov stuff to the .Rbuildignore created by `devtools::setup` rather than build our own?